testing/type_converters.go

```diff
@@ -1,4 +1,8 @@
-package raw_test
+package testing
+
+// String is a helper routine that allocates a new string value
+// to store v and returns a pointer to it.
+func String(v string) *string { return &v }
```

Comment:
Where is this used?  we should be able to use `github.String()` or `aws.String()` for most things.

Reply:
Bunch of tests use this. It didn't feel worth to refactor this other than moving this into a testing package.